targets/ar71xx-generic/profiles.mk

```diff
@@ -199,3 +199,10 @@ $(eval $(call GluonModel,ALL0315N,all0315n,allnet-all0315n))
 $(eval $(call GluonProfile,GLINET))
 $(eval $(call GluonModel,GLINET,gl-inet-6408A-v1,gl-inet-6408a-v1))
 $(eval $(call GluonModel,GLINET,gl-inet-6416A-v1,gl-inet-6416a-v1))
+
+## Meraki
+
+# Meraki MR12/MR62
+$(eval $(call GluonProfile,MR12))
+$(eval $(call GluonProfileFactorySuffix,MR12))
```

Comment:
Hmm, we should really try to provide an image which can be flashed over TFTP if possible. As the sysupgrade image is just flashed linearly by OpenWrt, have you tried just flashing it in one part through the bootloader?